src/main/java/gregtech/api/items/metaitem/MusicDiscStats.java

```diff
@@ -16,6 +16,8 @@
 
 public class MusicDiscStats implements IMusicDisc, IItemBehaviour {
 
+    public static final int SOUND_TYPE = 1050;
```

Comment:
Immediately noticing this. Why is this 1050 rather than 1010? Did I miss some important change?

Reply:
As I mentioned here. https://github.com/GregTechCEu/GregTech/pull/234#discussion_r756637012.

1010 is the type number is dedicated to `ItemRecord`.
The best way is a new type for MetaItem's Behaviours.

When hacking can be implemented as an extension, we should not break the oringinal logic.